lib/decision_review_v1/utilities/form_4142_processor.rb

```diff
@@ -49,23 +50,28 @@ def to_faraday_upload
       end
 
       def generate_metadata
-        veteran_full_name = @form['veteranFullName']
         address = @form['veteranAddress']
-        {
+        country_is_us = address['country'] == 'US' || address['countryCodeISO2'] == 'US'
```

Comment:
@tblackwe Made one adjustment to this after your approval, if you wouldn't mind reapproving, changing `country_is_us = address['country'] == 'US' || 'USA'` to the above, based on [address attributes in the Decision Review API](https://developer.va.gov/explore/api/decision-reviews/docs?version=current#:~:text=Schemas-,address,-contestableIssue). Thanks.